examples/async_req_rep.py

```diff
@@ -0,0 +1,39 @@
+import time
+
+from osbrain import run_agent
+from osbrain import run_nameserver
+
+
+def reply_late(agent, message):
+    time.sleep(1)
+    return 'Hello, Bob!'
+
+
+def process_reply(agent, message):
+    agent.log_info('Processed reply: %s' % message)
+
+
+def deaf(agent, message):
```

Comment:
This function is not used in this example.

Reply:
Good catch! Actually... the example is wrong, it includes the changes that the user is supposed to try by themselves (i.e.: the example should not contain the wait and on-error part).

Changing it. :blush: